homeassistant/components/modbus/climate.py

```diff
@@ -66,48 +68,69 @@
         vol.Optional(CONF_MAX_TEMP, default=35): cv.positive_int,
         vol.Optional(CONF_MIN_TEMP, default=5): cv.positive_int,
         vol.Optional(CONF_STEP, default=0.5): vol.Coerce(float),
+        vol.Optional(CONF_STRUCTURE, default=">f"): cv.string,
         vol.Optional(CONF_UNIT, default="C"): cv.string,
     }
 )
 
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Modbus Thermostat Platform."""
-    name = config[CONF_NAME]
```

Comment:
Please do not remove the declaration of these local variables. They make the code easier to read.